packages/vats/src/localchain.js

```diff
@@ -32,6 +35,9 @@ export const LocalChainAccountI = M.interface('LocalChainAccount', {
     .optional(M.pattern())
     .returns(AmountShape),
   executeTx: M.callWhen(M.arrayOf(M.record())).returns(M.arrayOf(M.record())),
+  monitorTransfers: M.callWhen(M.remotable('TransferTap')).returns(
+    M.remotable('Unregistrar'),
```

Comment:
What is an "Unregistrar"? Shouldn't this use the debug name from packages/vats/src/bridge-target.js?